shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/migration/AbstractMigrationITCase.java

```diff
@@ -126,15 +126,14 @@ protected void startMigrationOrderItem(final boolean withSchema) throws SQLExcep
     }
     
     protected void addMigrationProcessConfig() throws SQLException {
-        try {
-            proxyExecuteWithLog(migrationDistSQLCommand.getAddMigrationProcessConfig(), 0);
-        } catch (final SQLException ex) {
-            if ("58000".equals(ex.getSQLState()) || "HY000".equals(ex.getSQLState())) {
-                log.warn(ex.getMessage());
-                return;
+        if (ENV.getItEnvType() == ITEnvTypeEnum.NATIVE) {
+            try {
+                proxyExecuteWithLog("DROP MIGRATION PROCESS CONFIGURATION '/read'", 1);
+            } catch (final SQLException ex) {
```

Comment:
`'/read'` could just clean `READ` configuraion, `'/'` could be used to clean process configuration